src/node/rpc/commonhandlerregistry.h

```diff
@@ -245,7 +245,11 @@ namespace ccf
       install_with_auto_schema<GetCommit>(
         GeneralProcs::GET_COMMIT, handler_adapter(get_commit), Read);
       install_with_auto_schema<void, GetMetrics::Out>(
-        GeneralProcs::GET_METRICS, handler_adapter(get_metrics), Read, true);
+        GeneralProcs::GET_METRICS,
+        handler_adapter(get_metrics),
+        Read,
+        false,
```

Comment:
could you either comment here, or have named variables for `false` and `true`

Reply:
Added a comment :)